ide/editor.util/src/org/netbeans/lib/editor/util/FlyOffsetGapList.java

```diff
@@ -50,7 +50,7 @@ public FlyOffsetGapList(int initialCapacity) {
      *
      * @param elem element currently stored in the list.
      * @return raw offset of the element. It needs to be preprocessed
-     * by {@link #raw2RelOffset(int)} to become the real offset.
+     * by {@code #raw2RelOffset(int)} to become the real offset.
```

Comment:
i suppose that is because the method is private?

Reply:
yes this generate ok with @link but create a href that is invalid because of scope on final web page for a item you cannot find/browse.